File: CoPrime.java
import java.util.Scanner;

public class CoPrime {
    public static int gcd(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int a = sc.nextInt();

        System.out.print("Enter second number: ");
        int b = sc.nextInt();

        if (gcd(a, b) == 1)
            System.out.println("The numbers are Co-prime");
        else
            System.out.println("The numbers are NOT Co-prime");
    }
}

File: CoPrime.c
#include <stdio.h>

int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    int a, b;

    printf("Enter first number: ");
    scanf("%d", &a);

    printf("Enter second number: ");
    scanf("%d", &b);

    if (gcd(a, b) == 1)
        printf("The numbers are Co-prime");
    else
        printf("The numbers are NOT Co-prime");

    return 0;
}
